src/client/test/components/home/index.test.js

```diff
@@ -2,11 +2,11 @@ import React from 'react'
 import { shallow } from 'enzyme'
 import expect from 'expect'
 
-import Home from 'components/home'
+import Home from '../../../components/home'
```

Comment:
I think it's good to keep importing the components from `absolute path`-like so that you don't need to import the file with a lot of `..` 🤔 but it depends on you.

Reply:
Yes, it is cleaner if I use `absolute path`. I think `CPA` can use `.env` file and define `NODE_PATH` to help me